addon-test-support/pages/ember-table-body-page.js

```diff
@@ -0,0 +1,44 @@
+import PageObject, { collection, hasClass } from 'ember-classy-page-object';
+import { findAll } from 'ember-native-dom-helpers';
+import { findElement } from 'ember-classy-page-object/extend';
+import { getter } from 'ember-cli-page-object/macros';
+
+export default PageObject.extend({
+  scope: 'tbody',
+
+  /**
+   * List of rows in table body. Each of property/function in this collections is the property/func
+   * of a single row selected by using calling rows.eq(index).
+   */
+  rows: collection({
+    scope: 'tr',
+
+    /**
+     * List of all cells for the selected row.
+     */
+    cells: collection({
+      scope: 'td'
+    }),
+
+    /**
+     * Returns the height of selected row.
+     */
+    height: getter(function() {
```

Comment:
You shouldn't be using `getter`, you can do:

```js

get height() {

}
```

instead. I'm hoping to pull this functionality back upstream to ember-cli-page-object at some point.